### ISSUE TITLE
Spell checker fails to start

### ISSUE THREAD

**Issue by jankalfus** [Jason-Rev/vscode-spell-checker#157]
I'm getting the following error message in the output window for Code Spell Checker:
```
[Error - 09:31:50] Request textDocument/codeAction failed.
  Message: Request textDocument/codeAction failed with message: Cannot read property 'uri' of undefined
  Code: -32603 
```
When I try to open the spell checker configuration, VS Code displays the following error message:
`Unable to open 'Spell Checker Info': Request getConfigurationForDocument failed with message: [UriError]: Scheme contains illegal characters..`

I tried reinstalling the extension and at first run, I got the following error:
```
[Error - 09:40:10] Request textDocument/codeAction failed.
Error: Connection got disposed.
	at Object.dispose (C:\Users\Honza\.vscode\extensions\streetsidesoftware.code-spell-checker-1.5.0\node_modules\vscode-jsonrpc\lib\main.js:825:25)
	at Object.dispose (C:\Users\Honza\.vscode\extensions\streetsidesoftware.code-spell-checker-1.5.0\node_modules\vscode-languageclient\lib\client.js:57:35)
	at LanguageClient.handleConnectionClosed (C:\Users\Honza\.vscode\extensions\streetsidesoftware.code-spell-checker-1.5.0\node_modules\vscode-languageclient\lib\client.js:1864:38)
	at LanguageClient.handleConnectionClosed (C:\Users\Honza\.vscode\extensions\streetsidesoftware.code-spell-checker-1.5.0\node_modules\vscode-languageclient\lib\main.js:106:15)
	at closeHandler (C:\Users\Honza\.vscode\extensions\streetsidesoftware.code-spell-checker-1.5.0\node_modules\vscode-languageclient\lib\client.js:1852:18)
	at CallbackList.invoke (C:\Users\Honza\.vscode\extensions\streetsidesoftware.code-spell-checker-1.5.0\node_modules\vscode-jsonrpc\lib\events.js:71:39)
	at Emitter.fire (C:\Users\Honza\.vscode\extensions\streetsidesoftware.code-spell-checker-1.5.0\node_modules\vscode-jsonrpc\lib\events.js:135:36)
	at closeHandler (C:\Users\Honza\.vscode\extensions\streetsidesoftware.code-spell-checker-1.5.0\node_modules\vscode-jsonrpc\lib\main.js:221:26)
	at CallbackList.invoke (C:\Users\Honza\.vscode\extensions\streetsidesoftware.code-spell-checker-1.5.0\node_modules\vscode-jsonrpc\lib\events.js:71:39)
	at Emitter.fire (C:\Users\Honza\.vscode\extensions\streetsidesoftware.code-spell-checker-1.5.0\node_modules\vscode-jsonrpc\lib\events.js:135:36)
```
I have no idea what's wrong.

**Comment by Jason3S** [comment]
Thank you.

It is related to the update.

If you have a sample project. It would help a lot.

**Comment by Jason3S** [comment]
The spell checker was updated today.

There seems to be an issue related to Windows.

**Comment by jankalfus** [comment]
Oh, sorry for not including the version info, here it is:
- VSCode Version: Code 1.18.1 (929bacba01ef658b873545e26034d1a8067445e9, 2017-11-16T18:32:36.023Z)
- OS Version: Windows_NT x64 10.0.16299

**Comment by Jason3S** [comment]
I have put out a patch that rolls back most of the changes related to 1.5.0. I'll publish a new version when I get the issues for Windows fixed.

**Comment by jankalfus** [comment]
Looks like the "update" made it working again :) Thanks!

**Comment by Jason3S** [comment]
The "update" was just rolling back to an old version. I haven't had any time to fix the root issue of 1.15 yet. I hope to have some time tomorrow.

**Comment by Jason3S** [comment]
This should be fixed with 1.6.0
